runtime/compiler/ilgen/J9IlGeneratorMethodDetails.cpp

```diff
@@ -138,6 +165,7 @@ TR::IlGeneratorMethodDetails & IlGeneratorMethodDetails::create(
 
    TR_ResolvedJ9Method * j9method = static_cast<TR_ResolvedJ9Method *>(method);
 
+#if defined(JITSERVER_SUPPORT)
```

Comment:
The if and the else part of this #ifdef should be equivalent. We should leave only one of them.